src/components/ThumbnailsPanel/ThumbnailsPanel.js

```diff
@@ -208,26 +208,12 @@ class ThumbnailsPanel extends React.PureComponent {
 
   onWindowResize = () => {
     this.setState({
-      numberOfColumns: this.getNumberOfColumns(),
+      numberOfColumns: this.getNumberOfColumns(this.state.width),
```

Comment:
I think we can remove this function now, as we will get columns from width

Reply:
removed the `onWindowResize` fxn